test/expected/paths.out

```diff
@@ -103,6 +103,54 @@ where value->>'$ref' not like '#/components/parameters/rowFilter.products.%';
  {"$ref": "#/components/parameters/preferGet"}
 (10 rows)
 
+-- POST operation object
+-- shows the table name as tag
+select jsonb_pretty(get_postgrest_openapi_spec('{test}')->'paths'->'/products'->'post'->'tags');
```

Comment:
Hm, the `get` in `get_postgrest_openapi_spec` is a bit confusing. It conflates with the GET method. Could that function be named `postgrest_openapi_spec`?

Reply:
Yeah, it's a bit confusing. Done.